applications-frontend/vue.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    devServer: {
+        proxy: {
+            "/api": {
```

Comment:
The proxying is done internally using https://github.com/chimurai/http-proxy-middleware, so using "*" might allow you to consolidate these rules.

Reply:
can't, I don't want the frontend to pass /review into the backend. This happens on login redirect, or any sort of redirect.

Leaving /review out of this proxy list will correctly handle it in the devServer and do vue-router stuff

Reply:
Got it